packages/flame_network_assets/pubspec.yaml

```diff
@@ -0,0 +1,27 @@
+name: flame_network_assets
+description: Network assets support for Flame.
```

Comment:
nit: this will trigger the "package description too short" pub penalty. I think the min is 60 characters. not endorsing unnecessary verbosity but losing points is also bad